doc/dev/GETTING_STARTED.md

```diff
@@ -0,0 +1,172 @@
+# Getting started guide
+
+This is a getting started guide, aimed to people that are not used
+to C++ programming, configuring and compiling. If you are already used
+to such processes, you may be looking for our [build guide](BUILD.md).
+
+This guide will help you to compile your own version of F3D with required dependencies.
+Optional dependencies or covered in the build guide.
+
+ - If you are using [Linux](#Linux)
+ - If you are using [Windows](#Windows)
+ - If you are using [macOS](#macOS)
+
+## Linux
+
+Note: The following steps were tested with Ubuntu 22 and ArchLinux but
+should work for other OSes as stated, as long as listed packages are available.
+
+### Install dependencies
+
+First make sure your system is up-to-date,
+then install the required dependencies using the terminal.
+
+#### Debian/Ubuntu/Mint
+
+```
+sudo apt install build-essential git git-lfs cmake libvtk9-dev
+```
+
+#### Fedora/Centos/RedHat
+
+```
+sudo yum install make automake gcc gcc-c++ kernel-devel git git-lfs cmake vtk
+```
+
+#### Arch Linux
+
+```
+sudo pacman -S base-devel git git-lfs cmake vtk
+```
+
+### Compile F3D
+
+Open a terminal.
+
+```
+mkdir f3d
+cd f3d
+git clone https://github.com/f3d-app/f3d src
+mkdir build
+cd build
+cmake ../src
+make
+```
+
+You can then simply run F3D from the command line:
+
+```
+./bin/f3d
+```
+
+## Windows
+
+Note: The following steps were tested with Visual Studio Community 2022 using Windows 10, but should also work for other versions
+
+#### Install Dependencies
+ * Download and install [git bash for windows][gitforwindows]
+ * Download and install [cmake][cmake-download]
+ * Download and install [Visual Studio 2022 Community Edition][visual-studio]
+
+#### Recover F3D sources
+
+ * Open git bash
+
+```sh
+cd C:
+mkdir dev
+cd dev
+mkdir f3d
+cd f3d
+git clone https://github.com/f3d-app/f3d.git src
+mkdir build
+```
+
+#### Install dependencies using vcpkg
+
+ * Open git bash
+
+```sh
+cd C:/dev
+git clone https://github.com/Microsoft/vcpkg.git
+```
+
+ * Open cmd
+
+```sh
+cd C:\dev
```

Comment:
Not a big deal, but let's be consistent with the previous command

```suggestion
cd C:/dev
```

Reply:
cmd and git bash do not use the same syntax afaik.

Reply:
fair enough